bundler/lib/dependabot/bundler/file_updater/lockfile_updater.rb

```diff
@@ -33,11 +33,12 @@ def gemspec_sources
         end
 
         def initialize(dependencies:, dependency_files:,
-                       repo_contents_path: nil, credentials:)
+                       repo_contents_path: nil, credentials:, options: {})
```

Comment:
```suggestion
                       repo_contents_path: nil, credentials:, options:)
```

I've tried to follow the rule this is a required argument everywhere that isn't a top-level class, so this should be required here.